include/miral/miral/window_specification.h

```diff
@@ -149,6 +149,13 @@ class WindowSpecification
     auto exclusive_rect() -> mir::optional_value<mir::optional_value<mir::geometry::Rectangle>>&;
     ///@}
 
+    /// The D-bus service name and basename of the app's .desktop file
+    /// See http://standards.freedesktop.org/desktop-entry-spec/
```

Comment:
I don't follow the intention here. The equivalence of "D-bus service name" and "the basename" is only required for "Applications that support being launched by D-Bus". And is this optional attribute a good value to chose for `application_id()`?

Reply:
I'm just describing what it will be set to by XDG shell. Better if I change it to "Generally the D-bus service name..."?